charts/jenkins/pod-templates/nuxeo-package-lts-2021-node18.yaml.gotmpl

```diff
@@ -0,0 +1,47 @@
+- id: "nuxeo-package-lts-2021-node18"
+  name: "nuxeo-package-lts-2021-node18"
+  label: "jenkins-nuxeo-package-lts-2021-node18"
```

Comment:
Can we align https://github.com/nuxeo/platform-ci/commit/06c342cb2377e71a04fcc62f7238d7495a1773b6#diff-712269ffac481c64c79e5f9db19e1fd9d3d99abfa67b64fa46847e0fd40c623dR3 by the way to remove the `-`?
Bu then we probably need to change the Jenkinsfile in the repo(s) using it :/
Or maybe simpler, align everything on node-18 (or nodejs-18?).

Reply:
Let's change everything to `nuxeo-package-lts-2021-nodejs18` and `jenkins-nuxeo-package-lts-2021-nodejs18` to match the builder.